code/modules/admin/debug_verbs.dm

```diff
@@ -469,30 +437,10 @@ GLOBAL_PROTECT(AdminProcCallSpamPrevention)
 	var/dat = "<b>Contents of [key_name(M)]:</b><hr>"
 
 	var/list/L = M.get_contents()
-	for(var/t in L)
-		dat += "[t]<br>"
+	for(var/atom/A in L)
```

Comment:
This should only return atoms (items in fact). Type cast it outside to avoid unnecessary checks.